<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multicore</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="10,0">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string comment="www.kati.gr">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://europa.eu/index_el.htm&quot;&gt;&lt;img src=&quot;:/images/images/b.png&quot;/&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.digitalplan.gov.gr/portal/&quot;&gt;&lt;img src=&quot;:/images/images/a.png&quot;/&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.espa.gr/el/Pages/Default.aspx&quot;&gt;&lt;img src=&quot;:/images/images/c.png&quot;/&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Core 0</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QGridLayout" name="gridLayout">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QTableWidget" name="registersTable1">
                  <property name="font">
                   <font>
                    <family>Courier New</family>
                   </font>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="showGrid">
                   <bool>false</bool>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>80</number>
                  </attribute>
                  <attribute name="horizontalHeaderMinimumSectionSize">
                   <number>35</number>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Register</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Value</string>
                   </property>
                  </column>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCommandLinkButton" name="commandLinkButton">
                  <property name="text">
                   <string>Core Cycle</string>
                  </property>
                  <property name="shortcut">
                   <string notr="true">Space, Return, N</string>
                  </property>
                  <property name="autoRepeat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Code</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableView" name="codeList1">
                  <property name="horizontalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="dragDropMode">
                   <enum>QAbstractItemView::NoDragDrop</enum>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>10</width>
                    <height>10</height>
                   </size>
                  </property>
                  <property name="textElideMode">
                   <enum>Qt::ElideLeft</enum>
                  </property>
                  <property name="showGrid">
                   <bool>false</bool>
                  </property>
                  <property name="gridStyle">
                   <enum>Qt::NoPen</enum>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <property name="cornerButtonEnabled">
                   <bool>false</bool>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>180</number>
                  </attribute>
                  <attribute name="horizontalHeaderMinimumSectionSize">
                   <number>180</number>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderDefaultSectionSize">
                   <number>18</number>
                  </attribute>
                  <attribute name="verticalHeaderHighlightSections">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderMinimumSectionSize">
                   <number>18</number>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>L1 Cache</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableView" name="cacheViewC1">
                  <property name="font">
                   <font>
                    <family>Courier New</family>
                   </font>
                  </property>
                  <property name="showGrid">
                   <bool>true</bool>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderStretchLastSection">
                   <bool>false</bool>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Core 1</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QSplitter" name="splitter_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QTableWidget" name="registersTable2">
                  <property name="font">
                   <font>
                    <family>Courier New</family>
                   </font>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="showGrid">
                   <bool>false</bool>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>80</number>
                  </attribute>
                  <attribute name="horizontalHeaderMinimumSectionSize">
                   <number>35</number>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Register</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Value</string>
                   </property>
                  </column>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QCommandLinkButton" name="commandLinkButton_2">
                  <property name="text">
                   <string>Core Cycle</string>
                  </property>
                  <property name="autoRepeat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Code</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableView" name="codeList2">
                  <property name="showGrid">
                   <bool>false</bool>
                  </property>
                  <property name="gridStyle">
                   <enum>Qt::NoPen</enum>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <property name="cornerButtonEnabled">
                   <bool>false</bool>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>180</number>
                  </attribute>
                  <attribute name="horizontalHeaderMinimumSectionSize">
                   <number>180</number>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderDefaultSectionSize">
                   <number>18</number>
                  </attribute>
                  <attribute name="verticalHeaderHighlightSections">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderMinimumSectionSize">
                   <number>18</number>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>L1 Cache</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableView" name="cacheViewC2">
                  <property name="font">
                   <font>
                    <family>Courier New</family>
                   </font>
                  </property>
                  <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <property name="title">
           <string>System</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QCommandLinkButton" name="commandLinkButton_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>151</width>
             <height>39</height>
            </rect>
           </property>
           <property name="text">
            <string>System Cycle</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="totalCycles">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>30</y>
             <width>121</width>
             <height>16</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>30</y>
             <width>121</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Total system cycles</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>L2 Cache</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="memoryView">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load Code</string>
     </property>
     <addaction name="actionCore_0"/>
     <addaction name="actionCore_1"/>
    </widget>
    <addaction name="actionReset"/>
    <addaction name="menuLoad"/>
    <addaction name="actionReset_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCore_0">
   <property name="text">
    <string>Core 0</string>
   </property>
  </action>
  <action name="actionCore_1">
   <property name="text">
    <string>Core 1</string>
   </property>
  </action>
  <action name="actionReset_2">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
